src/pages/TCY/components/TCYCta.tsx

```diff
@@ -10,6 +10,26 @@ import { AssetIcon } from '@/components/AssetIcon'
 
 const margin = { base: 4, md: 0 }
 
+const CtaSkeleton = () => {
```

Comment:
suggestion: from a UX perspective I'm not sure about such skeleton, wouldn't we just be better off rendering a simple skeleton with the same visual structure, without any text, to keep things aligned visually and have everything the same skeleton-y?

Reply:
This text will never get rendered, it was just there to match the content that will eventually fill it in. I'll just hardcode the height.